cli/src/test/kotlin/de/deutschepost/sdm/cdlib/archive/ArchiveCommandIntegrationTest.kt

```
@@ -173,8 +173,7 @@ class ArchiveCommandIntegrationTest(@Value("\${artifactory-azure-identity-token}
         val repository = artifactory.repository(repoName)
         try {
             repository.delete(releaseName_build)
-        } catch (_: Exception) {
-        }
+        } catch (exception: Exception) { println("Error occurred while trying to delete the repository: ${exception.message}") }
```

To address the Sonar issue of an empty catch block, I followed the fix strategy by introducing a logging statement inside the catch block. The instructions specified using a logging mechanism consistent with the rest of the codebase. Since no specific logging library was mentioned in the used import statement list, I opted to use Kotlin's standard println function to log any exceptions that occur when attempting to delete the repository. This approach adds a simple and immediate way to see potential errors within system logs. Consequently, this change makes the system more robust by avoiding silent suppression of exceptions and assists in understanding failure motivations during the execution.<br><br>[Related Sonar Issue](http://localhost:9000/issues?open=15156638-4782-4aec-a774-3f852ff7c2ca)<br>[Related Sonar Rule](http://localhost:9000/coding_rules?open=kotlin:S108&rule_key=kotlin:S108)